rust/Cargo.toml

```diff
@@ -7,11 +7,14 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1"
-wasmtime = "8.0.0"
-wasmtime-wasi = "8.0.0"
-tokio = { version = "1.28.1", features = ["sync", "rt-multi-thread", "macros"] }
-rand = "0.8.5"
+wasmtime = ">= 8"
+wasmtime-wasi = ">= 8"
```

Comment:
@nilslice could use an eye on these. Loosened requirements for all deps. Unfortunately wasmtime doesn't follow semver but i think it's safe to just say we support 8 and above right now and keep an eye out for breaking changes in CI.